tests/unit/bokeh/plotting/test_figure.py

```diff
@@ -52,7 +52,18 @@
 #-----------------------------------------------------------------------------
 
 
-class Testfigure:
+class Test_figure:
+
+    def test_init(self) -> None:
+        f0 = bpf.figure(x_axis_type="linear")
+        assert isinstance(f0, bpf.figure)
+
+        with pytest.raises(ValueError, match=re.compile("linear")): # TODO: ValidationError
```

Comment:
Unless I am  mistaken, this will suffice for simple cases:
```suggestion
        with pytest.raises(ValueError, match="linear"): # TODO: ValidationError
```

Reply:
I thought `str` would mean literal matching, but apparently it's just a shortcut for `re.compile(...)`.